### PR TITLE
fix the design of the deposit and borrow button

### PR DESCRIPTION
**Pull request by LuisB0605**
IMG_021FF29F4605-1

### CODE REVIEW

File: components/atoms/BorrowButton.tsx [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 export const BorrowButton = () => {
     const navigation = useNavigation();
-    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} />
+    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} style={styles.button} />
 };
+
+const styles = StyleSheet.create({
+    button: {
+        borderTopLeftRadius: 0,
+        borderBottomLeftRadius: 0,
```
Comment (samhv) [inline components/atoms/BorrowButton.tsx:14]: Can we remove them?

File: components/atoms/BorrowButton.tsx [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+        borderBottomRightRadius: 20,
+        borderTopRightRadius: 20,
+        padding: 10,
```
Comment (samhv) [inline components/atoms/BorrowButton.tsx]: why do we need this padding?

File: components/atoms/BorrowButton.tsx [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
-    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} />
+    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} style={styles.button} />
 };
+
+const styles = StyleSheet.create({
+    button: {
+        borderTopLeftRadius: 0,
+        borderBottomLeftRadius: 0,
+        borderBottomRightRadius: 20,
+        borderTopRightRadius: 20,
```
Comment (samhv) [inline components/atoms/BorrowButton.tsx]: What are these numbers? Can we use somehow `STYLES` here?, something like `STYLES.buttonHeight/2` or something like that

File: components/atoms/DepositButton.tsx [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
-    return <Button title={"Deposit"} onPress={() => navigation.navigate("Deposit")} />
+    return <Button title={"Deposit"} onPress={() => navigation.navigate("Deposit")} style={styles.button} />
 };
+
+const styles = StyleSheet.create({
+    button: {
+        borderTopLeftRadius: 20,
+        borderBottomLeftRadius: 20,
+        borderBottomRightRadius: 0,
+        borderTopRightRadius: 0,
```
Comment (samhv) [inline components/atoms/DepositButton.tsx:14]: Can we remove them?

File: components/atoms/DepositButton.tsx [hunk 5]
```diff
@@ -1,8 +1,19 @@
...
+        padding: 10,
```
Comment (samhv) [inline components/atoms/DepositButton.tsx]: same comment that above, why do we need it? can we remove it? or can we use `STYLES` here?

**Review by samhv — CHANGES_REQUESTED** [review]
couple comments, let's address them before moving forward ;)

File: components/atoms/BorrowButton.tsx (hunk 1 above)
Comment (LuisB0605) [reply components/atoms/BorrowButton.tsx:14]: Actually we can remove the borderRadius with value 20, but we have to keep these ones.

File: components/atoms/BorrowButton.tsx (hunk 2 above)
Comment (LuisB0605) [reply components/atoms/BorrowButton.tsx]: I removed it

**Review by samhv — APPROVED** [review]
lgtm

**Pull request merged**